python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -437,6 +425,18 @@ def _update(self):
                     self.attempt_to_recover_unhealthy_nodes(now)
                 self.set_prometheus_updater_data()
 
+        # Update running nodes gauge
```

Comment:
Note: L406 - L426 update the list of alive nodes (i.e. `non_terminated_nodes`). This PR uses the updated list to update load metrics.